src/components/ConnectWallet/Wallets/ZapiOS.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <card-widget header="Here's how to connect Zap (iOS) to your Umbrel">
+    <div class="px-3 px-lg-4 pb-3">
+      <ol class="connect-wallet-steps">
+        <li class="connect-wallet-step">
+          Open the Zap app on your iPhone.
+        </li>
+        <li class="connect-wallet-step">
+          Click
+          <span class="font-weight-bold">"Connect"</span> on the welcome screen.
+        </li>
+        <li class="connect-wallet-step">
+          Scan this QR Code (click to enlarge)
+          <qr-code
+            :value="urls.lnd.restTor"
+            :size="300"
+            class="qr-image mt-2"
+            showLogo
+            @click="$emit('showQrModal', { wallet: 'Zap (iOS)', value: urls.lnd.restTor })"
```

Comment:
You also don't need the SSL cert here, you can strip it out for iOS like you do for Android.

Maybe we should remove this on the manager API end.

Reply:
>you can strip it out for iOS like you do for Android.

Will remove it.

>Maybe we should remove this on the manager API end.

But there are some wallets that might need it, aren't there? For the standard lndconnect links we should let the SSL cert attached, don't you think?

Reply:
Done in ee9a0c6.

Reply:
> You also don't need the SSL cert here, you can strip it out for iOS like you do for Android.
> 
> Maybe we should remove this on the manager API end.

Are you sure? If yes, then fine, but I was only talking about Zap Android. They have completely different code bases and therefore behave differentl in some situations. So if you did this just because of my previous comment, please double check if this is actually the case.

Reply:
@michaelWuensch Yeah I realised you just meant Android, I tested it out on iOS, thanks for clarifying though!

Reply:
So, does it work without the cert on iOS? Or should I revert the change.